enterprise/frontend/src/embedding-sdk/README.md

```diff
@@ -287,6 +287,41 @@ export default function App() {
 }
 ```
 
+### Embedding the collection browser
+
+With the Collection Browser, you can browse the items in your Metabase instance from your application.
+
+#### Parameters
+
+- **collectionId**: `number` – The numerical ID of the collection. You can find this ID in the URL when accessing a collection in your Metabase instance. For example, the collection ID in `http://localhost:3000/collection/1-my-collection` would be `1`. If no ID is provided, the collection browser will start at the root `Our analytics` collection.
```

Comment:
Anecdotally, I tested this component with `collectionId="root"` (string `"root"`). So it's not entirely correct that this has to be `number`, although, I agree that saying that this is only `number` makes sense from the API's standpoint.

It might be controversial, but I think at this point just say that the number `0` represents the root collection. We could map this in the SDK layer and translate it to `"root"` under the hood, since users shouldn't have to understand why we use string `"root"` for the root collection.

Reply:
I think using `0` would be way more consistent than using `root`

Reply:
I'll use `0` for now. I think a consistent numerical ID would be much easier to understand